Answers.md

```diff
@@ -0,0 +1,21 @@
+1. React JS is an open-source JavaScript library. It allows developers to create large     web applications. The diffence with React JS is that it uses a virtual DOM. It          completely manages the DOM for you. 
+
+2. The React component lifestyle are different methods used to override the code at particular times in the process to insert, use, and then remove from within the DOM.
+
+Constructor - is called before it is mounted.
+constructor(props) {

```

Comment:
yes. when used within the "classical context" a constructor will be used by the React lifecylce to setInitialState etc. But... this isn't a react lifecycle method. It's simple a method that comes with class. What we were looking for here is more like componentDidMount or componentWillReceiveProps